scripts/generator.py

```diff
@@ -59,6 +60,7 @@ def main():
     es_template.generate(nested, ecs_version, out_dir, args.mapping_settings)
     es_template.generate_legacy(flat, ecs_version, out_dir, args.template_settings, args.mapping_settings)
     beats.generate(nested, ecs_version, out_dir)
+    rules_generator.generate(flat, ecs_version, out_dir)
```

Comment:
Will we want to include the also add the generated rules file? Not seeing the `generated/rules/` directory included right now.